distro/superbuild/cmake/externals.cmake

```diff
@@ -224,9 +242,18 @@ endif()
 
 ###############################################################################
 # PythonQt
+set(PythonQt_REPO)
+set(PythonQt_TAG)
+if(${DD_QT_VERSION} VERSION_GREATER "4")
+  set(PythonQt_REPO https://github.com/patmarion/PythonQt.git)
+  set(PythonQt_TAG fix-compile-error-on-qt5.8)
```

Comment:
yes, eventually we will get this in.  upstream PythonQt is actually a svn repo.  The CommonToolkit group manages some unofficial patched branches on Github.  I think this is already fixed upstream in svn but I'm not sure if CTK's branches are up to date or not.  I will try to look into it.  In the meantime, I have no problem referencing this branch.